lesson_01/homework01.py

```diff
@@ -0,0 +1,120 @@
+"""

```

Comment:
Please create for each hometask separate branch.
 You need in git to return to branch main ( git checkout main)-> update main branch ( git pull origin)  and then create new for Hometask #3 ( git checkout -b <name_of_branch_for_hometask_3>)